rental.rb

```diff
@@ -0,0 +1,11 @@
+class Rental
+  attr_accessor :date, :person, :book
+
+  def initialize(date, person, book)
+    @date = date
+    @person = person
+    person.rentals << self
```

Comment:
- [ ] Great job for making sure that a rental belongs to a person :+1: as requested in the requirements. Now let's make sure that a person has many rentals by adding the `add_rental` method in the Person's class. Are you doubting what to do? No worries, let's kindly explore this [lesson](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship)